[
   {
      "author_association" : "CONTRIBUTOR",
      "body" : "<!--e57a25ab6845829454e8d69fc972939a-->\n\nThe following sections might be updated with supplementary metadata relevant to reviewers and maintainers.\n\n<!--174a7506f384e20aa4161008e828411d-->\n### Conflicts\nReviewers, this pull request conflicts with the following ones:\n\n* #20120 (net, rpc, test, bugfix: update GetNetworkName, GetNetworksInfo, regression tests by jonatack)\n* #19168 (Refactor: Improve setup_clean_chain semantics by fjahr)\n* #18766 (Disable fee estimation in blocksonly mode (by removing the fee estimates global) by darosior)\n* #17580 (refactor: Add ALLOW_LIST flags and enforce usage in CheckArgFlags by ryanofsky)\n\nIf you consider this pull request important, please also help to review the conflicting pull requests. Ideally, start with the one that should be merged first.",
      "created_at" : "2020-10-27T12:26:16Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/20254#issuecomment-717208517",
      "id" : 717208517,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20254",
      "node_id" : "MDEyOklzc3VlQ29tbWVudDcxNzIwODUxNw==",
      "performed_via_github_app" : null,
      "updated_at" : "2020-10-27T12:26:16Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/717208517",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/39886733?v=4",
         "events_url" : "https://api.github.com/users/DrahtBot/events{/privacy}",
         "followers_url" : "https://api.github.com/users/DrahtBot/followers",
         "following_url" : "https://api.github.com/users/DrahtBot/following{/other_user}",
         "gists_url" : "https://api.github.com/users/DrahtBot/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/DrahtBot",
         "id" : 39886733,
         "login" : "DrahtBot",
         "node_id" : "MDQ6VXNlcjM5ODg2NzMz",
         "organizations_url" : "https://api.github.com/users/DrahtBot/orgs",
         "received_events_url" : "https://api.github.com/users/DrahtBot/received_events",
         "repos_url" : "https://api.github.com/users/DrahtBot/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/DrahtBot/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/DrahtBot/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/DrahtBot"
      }
   },
   {
      "author_association" : "CONTRIBUTOR",
      "body" : " Ì¶pÌ¶lÌ¶sÌ¶ Ì¶nÌ¶oÌ¶tÌ¶eÌ¶ Ì¶tÌ¶hÌ¶eÌ¶ Ì¶iÌ¶2Ì¶pÌ¶ Ì¶pÌ¶rÌ¶oÌ¶xÌ¶yÌ¶ Ì¶tÌ¶aÌ¶kÌ¶eÌ¶sÌ¶ Ì¶aÌ¶dÌ¶dÌ¶rÌ¶eÌ¶sÌ¶sÌ¶eÌ¶sÌ¶ Ì¶iÌ¶nÌ¶ Ì¶tÌ¶hÌ¶eÌ¶ Ì¶fÌ¶oÌ¶rÌ¶mÌ¶ Ì¶yÌ¶fÌ¶sÌ¶vÌ¶sÌ¶yÌ¶4Ì¶6Ì¶7Ì¶mÌ¶tÌ¶5Ì¶xÌ¶aÌ¶fÌ¶aÌ¶qÌ¶7Ì¶zÌ¶aÌ¶uÌ¶kÌ¶kÌ¶jÌ¶yÌ¶zÌ¶eÌ¶hÌ¶vÌ¶mÌ¶eÌ¶wÌ¶4Ì¶4Ì¶5Ì¶yÌ¶aÌ¶aÌ¶eÌ¶jÌ¶vÌ¶rÌ¶wÌ¶pÌ¶kÌ¶5Ì¶3Ì¶aÌ¶cÌ¶eÌ¶jÌ¶qÌ¶.Ì¶iÌ¶2Ì¶pÌ¶\r\nÌ¶wÌ¶iÌ¶tÌ¶hÌ¶oÌ¶uÌ¶tÌ¶ Ì¶.Ì¶bÌ¶3Ì¶2Ì¶.Ì¶ Ì¶AÌ¶FÌ¶AÌ¶IÌ¶KÌ¶\r\n@vasild is ur node up? I was unable to reach from remote or is it by now just local?\r\nAnd by what directory is it published, AFAIK one has to import that directory and address mappings before u can use an address. ",
      "created_at" : "2020-10-27T14:15:02Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/20254#issuecomment-717273719",
      "id" : 717273719,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20254",
      "node_id" : "MDEyOklzc3VlQ29tbWVudDcxNzI3MzcxOQ==",
      "performed_via_github_app" : null,
      "updated_at" : "2020-10-27T15:50:50Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/717273719",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/34917548?v=4",
         "events_url" : "https://api.github.com/users/Saibato/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Saibato/followers",
         "following_url" : "https://api.github.com/users/Saibato/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Saibato/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Saibato",
         "id" : 34917548,
         "login" : "Saibato",
         "node_id" : "MDQ6VXNlcjM0OTE3NTQ4",
         "organizations_url" : "https://api.github.com/users/Saibato/orgs",
         "received_events_url" : "https://api.github.com/users/Saibato/received_events",
         "repos_url" : "https://api.github.com/users/Saibato/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Saibato/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Saibato/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Saibato"
      }
   },
   {
      "author_association" : "CONTRIBUTOR",
      "body" : "Yes, it is up. From another computer:\r\n\r\n```\r\n2020-10-27T14:31:09Z New outbound peer connected: version: 70015, blocks=654455, peer=0, peeraddr=yfsvsy467mt5xafaq7zaukkjyzehvmew445yaaejvrwpk53acejq.b32.i2p:31872 (full-relay)\r\n```\r\n\r\nThe `.b32` must be present. See \"B32\" in https://i2pplus.com/glossary/. The port seems to be irrelevant. I guess we can skip it altogether.",
      "created_at" : "2020-10-27T14:39:03Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/20254#issuecomment-717290115",
      "id" : 717290115,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20254",
      "node_id" : "MDEyOklzc3VlQ29tbWVudDcxNzI5MDExNQ==",
      "performed_via_github_app" : null,
      "updated_at" : "2020-10-27T14:46:50Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/717290115",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/266751?v=4",
         "events_url" : "https://api.github.com/users/vasild/events{/privacy}",
         "followers_url" : "https://api.github.com/users/vasild/followers",
         "following_url" : "https://api.github.com/users/vasild/following{/other_user}",
         "gists_url" : "https://api.github.com/users/vasild/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/vasild",
         "id" : 266751,
         "login" : "vasild",
         "node_id" : "MDQ6VXNlcjI2Njc1MQ==",
         "organizations_url" : "https://api.github.com/users/vasild/orgs",
         "received_events_url" : "https://api.github.com/users/vasild/received_events",
         "repos_url" : "https://api.github.com/users/vasild/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/vasild/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/vasild/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/vasild"
      }
   },
   {
      "author_association" : "CONTRIBUTOR",
      "body" : "> Yes, it is up. From another computer:\r\n\r\nthx, maybe i must update my version i use  i2pd 2.29.0 ?\r\n\r\nediit@saibato\r\n@vasild \r\nWas able to connect  :+1: \r\nI had forgot to create a fixed i2p client tunnel in i2pd for port 31872. \r\n\r\nconcept tACK \r\n\r\ncmd line  `-connect=yfsvsy467mt5xafaq7zaukkjyzehvmew445yaaejvrwpk53acejq.b32.i2p:31872  `then worked. ",
      "created_at" : "2020-10-27T14:51:58Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/20254#issuecomment-717298847",
      "id" : 717298847,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20254",
      "node_id" : "MDEyOklzc3VlQ29tbWVudDcxNzI5ODg0Nw==",
      "performed_via_github_app" : null,
      "updated_at" : "2020-10-27T17:26:34Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/717298847",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/34917548?v=4",
         "events_url" : "https://api.github.com/users/Saibato/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Saibato/followers",
         "following_url" : "https://api.github.com/users/Saibato/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Saibato/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Saibato",
         "id" : 34917548,
         "login" : "Saibato",
         "node_id" : "MDQ6VXNlcjM0OTE3NTQ4",
         "organizations_url" : "https://api.github.com/users/Saibato/orgs",
         "received_events_url" : "https://api.github.com/users/Saibato/received_events",
         "repos_url" : "https://api.github.com/users/Saibato/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Saibato/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Saibato/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Saibato"
      }
   },
   {
      "author_association" : "CONTRIBUTOR",
      "body" : "> Was able to connect\r\n\r\nExcellent!\r\n\r\nHowever I got:\r\n\r\n```\r\npid 82450 (i2pd), jid 0, uid 255: exited on signal 10\r\npid 19775 (i2pd), jid 0, uid 255: exited on signal 6\r\npid 19909 (i2pd), jid 0, uid 255: exited on signal 6\r\npid 20207 (i2pd), jid 0, uid 255: exited on signal 6\r\n```\r\nI stopped it until I figure out what's going on.",
      "created_at" : "2020-10-27T17:37:38Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/20254#issuecomment-717408597",
      "id" : 717408597,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20254",
      "node_id" : "MDEyOklzc3VlQ29tbWVudDcxNzQwODU5Nw==",
      "performed_via_github_app" : null,
      "updated_at" : "2020-10-27T17:37:38Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/717408597",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/266751?v=4",
         "events_url" : "https://api.github.com/users/vasild/events{/privacy}",
         "followers_url" : "https://api.github.com/users/vasild/followers",
         "following_url" : "https://api.github.com/users/vasild/following{/other_user}",
         "gists_url" : "https://api.github.com/users/vasild/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/vasild",
         "id" : 266751,
         "login" : "vasild",
         "node_id" : "MDQ6VXNlcjI2Njc1MQ==",
         "organizations_url" : "https://api.github.com/users/vasild/orgs",
         "received_events_url" : "https://api.github.com/users/vasild/received_events",
         "repos_url" : "https://api.github.com/users/vasild/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/vasild/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/vasild/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/vasild"
      }
   },
   {
      "author_association" : "CONTRIBUTOR",
      "body" : "> I stopped it until I figure out what's going on.\r\n\r\ntyi, while testing i also direct connected with netcat and connect-proxy without the bitcoin client and hammered on my keyboard, happy fuzzing! Maybe something @practicalswift might want to look in? \r\n\r\n",
      "created_at" : "2020-10-27T17:53:05Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/20254#issuecomment-717417464",
      "id" : 717417464,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20254",
      "node_id" : "MDEyOklzc3VlQ29tbWVudDcxNzQxNzQ2NA==",
      "performed_via_github_app" : null,
      "updated_at" : "2020-10-27T17:53:28Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/717417464",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/34917548?v=4",
         "events_url" : "https://api.github.com/users/Saibato/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Saibato/followers",
         "following_url" : "https://api.github.com/users/Saibato/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Saibato/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Saibato",
         "id" : 34917548,
         "login" : "Saibato",
         "node_id" : "MDQ6VXNlcjM0OTE3NTQ4",
         "organizations_url" : "https://api.github.com/users/Saibato/orgs",
         "received_events_url" : "https://api.github.com/users/Saibato/received_events",
         "repos_url" : "https://api.github.com/users/Saibato/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Saibato/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Saibato/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Saibato"
      }
   }
]
